ArdentHQ/platform-sdk
# feat: implement username register/resign transaction types

Closes: https://app.clickup.com/t/86drv5ncc

**Testing:**
- Pull this branch down
- Build the app
   - run `pnpm purge` to clear caches or `pnpm store prune`
   - `pnpm build:release` in the root of the platform sdk to ensure all packages get updated
- Checkout the `feat/mainsail` branch in arkvault
- Modify function [name](https://github.com/ArdentHQ/arkvault/blob/6cf06f54967612cf14f79e29357f3167b3db0346/src/domains/transaction/components/DelegateRegistrationForm/DelegateRegistrationForm.tsx#L67) to `signUsernameRegistration` or `signUsernameResignation` to test respective functionality 
- Run arkvault test server
   - `rm -rf node_modules` and  `pnpm i` to reinstall the packages
   - `pnpm dev`
- Set fee to 25 and proceed
- Check the explorer to see result of your action: https://explorer-demo.mainsailhq.com/
